app/src/main/res/values-en/strings.xml

```diff
@@ -835,30 +835,30 @@ However, before uploading your changes, the app checks with a &lt;a href=\"https
     <string name="quest_crossing_type_marked">Marked</string>
     <string name="quest_crossing_type_unmarked">Without road markings</string>
 
-    <string name="quest_cycleway_title2">"Is there a cycleway here? What type?"</string>
```

Comment:
note that it includes asking also about contraflow, contraflow lanes, bike paths, cycling allowed on sidewalks etc

Reply:
I think you're referring to line 839?  Would "Is this still the cycling situation here?" be better?

Reply:
> Would "Is this still the cycling situation here?" be better?

I think so but I am so NOT a native speaker

Reply:
Either way I think would be fine, so if the cycling version is more clear to non-native speakers, that works for me.

Reply:
Updated.